sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -530,6 +540,25 @@ puts that `command_graph` into an undefined state. Any further operations
 performed on a `command_graph` in this state will result in undefined
 behavior.
 
+===== Assume-Buffer-Outlives-Graph Property [[assume-buffer-outlives-graph-property]]
+The `property::graph::assume_buffer_outlives_graph` property disables
+restrictions on using buffers in a `command_graph` and can be passed to a
```

Comment:
It might be useful to add a link to the buffer-limitation section as readers have not read about it yet. (We are in Section 7.6.2 and "Buffer limitations" comes in 7.11.2)